AdvancedLifeCycleManager.java

```diff
@@ -46,26 +46,32 @@ public AdvancedLifeCycleManager() {
 		logger.finest("discardThreshold=" + discardThreshold);
 	}
 
-        boolean isFailure(byte state) {
-                switch (state) {
-                case PduConstants.DELIVERED:
-                        return false;
-                case PduConstants.ACCEPTED:
-                        return false;
-                default:
-                        return true;
-                }
-        }
-
-        void prepDeliveryReceipt(MessageState m, SubmitSM p) {
-                logger.info("Delivery Receipt requested");
-                smsc.prepareDeliveryReceipt(p, m.getMessage_id(), m.getState(), 1, 1, m.getErr());
-        }
+	boolean isFailure(byte state) {
+		switch (state) {
+		case PduConstants.DELIVERED:
+				return false;
+		case PduConstants.ACCEPTED:
+				return false;
+		default:
+				return true;
+			}
+	}
+
+	void prepDeliveryReceipt(MessageState m, SubmitSM p) {
+		logger.info("Delivery Receipt requested");
+		smsc.prepareDeliveryReceipt(p, m.getMessage_id(), m.getState(), 1, 1, m.getErr());
+	}
 
 	public MessageState setState(MessageState m) {
 		// Should a transition take place at all?
+		logger.info("Using DeterministicLifeCycleManager");

```

Comment:
Wrong messages.. this is, in fact, _not_ 'DeterministicLifeCycleManager', but 'AdvancedLifeCycleManager'.